app/src/organisms/DropTipWizard/index.tsx

```diff
@@ -347,14 +357,25 @@ export const DropTipWizardComponent = (
 
     return retractAllAxesAndSavePosition()
       .then(currentPosition => {
-        if (currentPosition != null) {
+        const addressableAreaFromConfig = getAddressableAreaFromConfig(
+          addressableArea,
+          deckConfig,
+          instrumentModelSpecs.channels,
+          robotType
+        )
+
+        if (
+          currentPosition != null &&
+          (robotType !== FLEX_ROBOT_TYPE || addressableAreaFromConfig != null)
```

Comment:
```suggestion
          (currentPosition != null && addressableAreaFromConfig != null)
```
I'm guessing this is just a typo? We want to run the command on flexes as well (and it does work just fine if we remove this check!)

Reply:
`robotType !== FLEX_ROBOT_TYPE` bypasses the `addressableAreaFromConfig` check for OT-2. the intent is to be conservative with OT-2 since deck config is a Flex feature. the getAddressableAreaFromConfig helper should be irrelevant to OT-2 since there's a single configuration.

if it's cleaner, we can remove the robot type check. at least on 7.1 I confirmed that OT-2 returns a deck config that is compatible with the output of deck location select.